eng/pipelines/scripts/Extract-Assets.ps1

```diff
@@ -0,0 +1,66 @@
+param(
+  [Parameter(Mandatory=$true)][string] $SourcePath,
+  [Parameter(Mandatory=$true)][string] $TargetPath,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+if ($null -ne $properties) {
+  Write-Host "Unexpected extra parameters: $properties."
+  exit 1
+}
+
+Add-Type -AssemblyName System.IO.Compression.FileSystem
+
+function Create-PackageDir() {
+  param( 
+    [string] $FileName
+  )
+  $extractionPath = Join-Path -Path $TargetPath -ChildPath ($FileName + "_dir")
+  New-Item -ItemType Directory -Force -Path $extractionPath | Out-Null
+  return $extractionPath
+}
+
+function Extract-Zip() {
+  param(
+    [System.IO.FileInfo] $FileInfo
+  )
+  $extractionPath = Create-PackageDir -FileName $FileInfo.Name
+  Write-Host "Extracting $($FileInfo.Name)"
+  [System.IO.Compression.ZipFile]::ExtractToDirectory($FileInfo.FullName, $extractionPath)
+}
+
+function Extract-TarGz() {
+  param(
+    [System.IO.FileInfo] $FileInfo
+  )
+  $extractionPath = Create-PackageDir -FileName $FileInfo.Name
+  Write-Host "Extracting $($FileInfo.Name)"
+  tar -xf $FileInfo.FullName -C $extractionPath
```

Comment:
```suggestion
  tar -xzf $FileInfo.FullName -C $extractionPath
```

We should be specifying the compression type of the archive

Reply:
I've pushed this change and am running another validation build for it.